wire-ios-data-model/Source/Model/User/ZMUser.m

```diff
@@ -653,7 +653,7 @@ + (ZMUser *)unboxSelfUserFromContextUserInfo:(NSManagedObjectContext *)moc
     return boxed.selfUser;
 }
 
-+ (void)boxSelfUser:(ZMUser *)selfUser inContextUserInfo:(NSManagedObjectContext *)moc
++ (void)boxSelfUser:(ZMUser *)selfUser inContextUserInfo:(NSManagedObjectContext *)moc // TODO: relevant for import?
```

Comment:
not sure, I would say no, but what's your question exactly?

Reply:
Also this I wanted to remove this TODO before you start reviewing. Already deleted now.